src/components/heading/heading.spec.js

```diff
@@ -120,6 +155,13 @@ describe("Heading", () => {
     });
   });
 
+  describe("no pills", () => {
+    it("returns no pills wrapper", () => {
+      const wrapper = mount(<Heading title="test" />);
+      expect(wrapper.find(StyledHeadingPills).length).toEqual(0);
```

Comment:
you could check if the component exists:
`expect(wrapper.find(StyledHeadingPills).exists()).toBe(false)`

Reply:
Will change